av_imu_launch/config/av_gx5_15.yml

```diff
@@ -0,0 +1,15 @@
+microstrain_inertial_driver:
+  ros__parameters:
+    # You should change this section of config to match your setup
+    port : '/dev/imu-front'
+    baudrate : 230400
+
+    # This will cause the node to convert any NED measurements to ENU
```

Comment:
Indeed, the pre-commit (Which I got from [elsewhere](https://github.com/ros-controls/ros2_control/blob/master/.pre-commit-config.yaml)) is a little over-zealous right now.
I would add any original source code (e.g. from a fork) to be ignored by the .pre-commit (We're not trying to fix upstream source code, and we don't want to change it in our fork to avoid any conflicts). I forked a bunch of repos at the start for testing which may no longer need the source code, so that also needs doing.

For this particular pre-commit typo, I'll just reword the comment as you suggest. I would like to run pre-commit for all our work when applicable with few exceptions (e.g. I added [*.dae to be ignored](https://github.com/ipab-rad/car_description/pull/1/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R131) in the car-description repo).

Reply:
Changed my mind :sweat_smile: I'm adding it to the list of codespell ignored words since we "could" (Given it's an IMU launch repo) use the acronym NED with reasonable context and it will be picked up _again_.